### ISSUE TITLE
Add option to prevent renaming id namespace 

### ISSUE THREAD

**Issue by hanc2006** [iest/babel-plugin-inline-svg#15]
I didn't find a way to prevent the renaming id namespace. It would be nice to be able to disable this feature with a new configuration entry :)

**Comment by moelders** [comment]
Hello @hanc2006, @iest ,
I have been working on this issue because I had the same problem, I have seen that the prefix option comes with the plugin `prefixIds`, so, it is not necessary to force the prefix ([documentation](https://github.com/moelders/babel-plugin-inline-svg/tree/1.1.1#2-namespaces-ids-to-prevent-conflicts)).

You can see the changes and documentation in my fork: [_moelders/babel-plugin-inline-svg_](https://github.com/moelders/babel-plugin-inline-svg).

As I have no permissions to publish, and this repository seems to have no maintenance, the plugin can be installed via GitHub:

```json
{
  "devDependencies": {
    "babel-plugin-inline-svg": "moelders/babel-plugin-inline-svg#semver:1.1.1"
  }
}
```

**Comment by iest** [comment]
@moelders if you open a PR I'd be happy to check it out — from a brief look at your branch it's only some of adidas-specific eslint rules that would need to change.

I've been meaning to do a general cleanup of this repo for a while, so that would be a great place to start 🙌

**Comment by moelders** [comment]
Hello @iest,
I am sorry for my late reply.
As I use it in @adidas organization, I added adidas Eslint rules, however, it is not the focus of the changes. The problem was that the namespace was always renamed and it creates inconsistencies. So, the `prefixIds` should not force it if it is empty.
Let me know if you want to add my changes so I can open a PR to the main repository.

**Comment by iest** [comment]
@moelders please feel free to open a PR! I will happily help to get this change in

**Comment by moelders** [comment]
Some days late but the PR is open: #19.

**Comment by iest** [comment]
This issue was solved in #22
